.github/workflows/test.yaml

```diff
@@ -50,3 +50,23 @@ jobs:
         run: npm run format-check --prefix assets
       - name: Run assets tests
         run: npm test --prefix assets
+  # Verify that the app build builds as expected
+  app:
+    runs-on: macos-11
+    steps:
+      - uses: actions/checkout@v2
+      - run: brew install elixir
```

Comment:
I believe this is gonna be pretty slow unless cached too (though caching might not be super trivial)

Reply:
Yeah, took 53s and I don't think we can easily cache it. `erlef/setup-beam` doesn't support macOS though :< Will see how the build takes in total with caching, but alternatively we could run it only on `main` or maybe run it only when certain files change.